src/main/java/uk/gov/hmcts/sptribs/document/model/CaseworkerCICDocument.java

```diff
@@ -74,6 +74,6 @@ public boolean isDocumentValid(String validExtensions) {
 
     @JsonIgnore
     public boolean isValidBundleDocument() {
-        return isValidDocument(this.documentLink.getFilename(),"pdf,txt,xlsx,docx,doc,xls");
+        return isValidDocument(this.documentLink.getFilename(),"pdf,txt,xlsx,docx,doc,xls,jpg,jpeg");
```

Comment:
I think it is better to include other image formats like **tif, tiff, bmp and png** as well to support the user to upload any format of images.

Reply:
sure, can add files from this file https://github.com/hmcts/em-stitching-api/pull/2297/files